File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/04. Trekking Mania/04. Trekking Mania.js
function trekkingMania(arg) {
   let groups = Number(arg.shift())
   let all_people = 0
   let [g1, g2, g3, g4, g5] = [0, 0, 0, 0, 0]

   for (let i = 0; i < groups; i++) {
      people = Number(arg.shift())
      all_people += people

      if (people <= 5) {
         g1 += people
      } else if (people >= 6 && people <= 12) {
         g2 += people
      } else if (people >= 13 && people <= 25) {
         g3 += people
      } else if (people >= 26 && people <= 40) {
         g4 += people
      } else {
         g5 += people
      }
   }

   console.log(`${(g1 / all_people * 100).toFixed(2)}%`)
   console.log(`${(g2 / all_people * 100).toFixed(2)}%`)
   console.log(`${(g3 / all_people * 100).toFixed(2)}%`)
   console.log(`${(g4 / all_people * 100).toFixed(2)}%`)
   console.log(`${(g5 / all_people * 100).toFixed(2)}%`)
}

// trekkingMania([
//    10,
//    10,
//    5,
//    1,
//    100,
//    12,
//    26,
//    17,
//    37,
//    40,
//    78,
// ])

File: Python Exams/01. Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/04. Trekking Mania/04. Trekking Mania.py
groups = int(input())
all_people = 0
g1, g2, g3, g4, g5 = 0, 0, 0, 0, 0

for _ in range(groups):
   people = int(input())
   all_people += people 

   if people <= 5:
      g1 += people
   elif people >= 6 and people <= 12:
      g2 += people
   elif people >= 13 and people <= 25:
      g3 += people
   elif people >= 26 and people <= 40:
      g4 += people
   else:
      g5 += people


print(f'{(g1 / all_people * 100):.2f}%')
print(f'{(g2 / all_people * 100):.2f}%')
print(f'{(g3 / all_people * 100):.2f}%')
print(f'{(g4 / all_people * 100):.2f}%')
print(f'{(g5 / all_people * 100):.2f}%')


